src/filters/dates/relative/relativeDates.constants.ts

```diff
@@ -0,0 +1,57 @@
+const TODAY = 'today';
+const TOMORROW = 'tomorrow';
+
+const RELATIVE_ADVERB = {
+    ANY: `${TODAY}|${TOMORROW}`,
+    TODAY: `${TODAY}`,
+    TOMORROW: `${TOMORROW}`,
+};
+
+const RELATIVE_PREPOSITIONS = '((in|after) )?';
+const RELATIVE_POSTPOSTIONS = ' (from now|later|after)';
```

Comment:
This is wrong
Can we write it like this so it is clearer

```suggestion
const RELATIVE_POSTPOSTIONS = '((from now)|(later)|(after))';
```

Reply:
What do you mean by "wrong"? It is passing tests.

Reply:
Initially I thought it was wrong, then I realised I misread the brackets. But I didn't delete the original message properly it seems

Reply:
I think for this comment you literally just need to click on accept. Don't need code change

Reply:
I don't think the extra brackets are needed though, does it really add clarity?

Reply:
Yeah the space is more obvious now